src/app/layout.tsx

```diff
@@ -13,7 +17,53 @@ export default function RootLayout({
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
       </Head>
-      <body>{children}</body>
+      <body>
+        <div aria-label="Official government website">
+          <div className="nj-banner">
+            <div className="nj-banner__header">
+              <div className="grid-container">
+                <div className="nj-banner__inner">
+                  <div>
+                    <Image
+                      className="nj-banner__header-seal"
+                      src={njStateSeal}
+                      alt="NJ flag"
+                    ></Image>
+                  </div>
+                  <div className="grid-col-fill">
+                    <a href="https://nj.gov">Official Site of the State of New Jersey</a>
+                  </div>
+                  <div className="grid-col-auto">
+                    <div className="text-white">
+                      <ul>
+                        <li>Governor Phil Murphy &bull; Lt. Governor Tahesha Way</li>
+                        <li>
+                          <a href="https://nj.gov/subscribe/" target="_blank">
```

Comment:
```suggestion
                          <a href="https://nj.gov/subscribe/" target="_blank" aria-label="Get Updates (opens in new window)">
```

Links that open in a new window/tab should announce to screen readers